Since glib-2.32, this is a common error:
  In file included from rsa_nss.c:27:0:
  /usr/include/glib-2.0/glib/gmain.h:21:2: error: #error "Only <glib.h> can be included directly."
Let's fix it :)

diff -Nur pidgin-encryption-3.1.orig/rsa_nss.c pidgin-encryption-3.1/rsa_nss.c
--- pidgin-encryption-3.1.orig/rsa_nss.c	2010-04-25 20:53:46.000000000 -0500
+++ pidgin-encryption-3.1/rsa_nss.c	2012-05-01 22:58:18.033710803 -0500
@@ -24,7 +24,7 @@
 #include <debug.h>
 #include <gtkdialogs.h>
 
-#include "glib/gmain.h"
+#include <glib.h>
 
 #include <string.h>
 #include <assert.h>
